content/en/docs/tasks/debug/debug-application/debug-pods.md

```diff
@@ -69,6 +69,34 @@ There are three things to check:
 * Try to manually pull the image to see if the image can be pulled. For example,
   if you use Docker on your PC, run `docker pull <image>`.
 
+
+#### My pod stays terminating
+
+If a Pod is stuck in the `Terminating` state, it means that a deletion has been
+issued for the Pod, but the control plane is unable to delete the Pod object.
+
+This typically happens if the Pod has a [finalizer](/docs/concepts/overview/working-with-objects/finalizers/)
+and there is an [admission webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
+installed in the cluster that prevents the control plane from removing the
+finalizer.
+
+To identify this scenario, check if your cluster has any
+ValidatingWebhookConfiguration or MutatingWebhookConfiguration that target
+`UPDATE` operations for `pods` resources.
+
+If the webhook is provided by a third-party:
+- Make sure you are using the latest version.
+- Disable the webhook for `UPDATE` operations.
+- Report an issue with the corresponding provider.
+
+If you are the author of the webhook:
+- For a mutating webhook, make sure it never changes immutable fields on
+  `UPDATE` operations. Typically, any container changes are not allowed.
```

Comment:
```suggestion
  `UPDATE` operations.
```
In the https://kubernetes.io/blog/2023/05/12/in-place-pod-resize-alpha/, this is not true anymore.

Reply:
I guess I can change it by "most", instead of "any".
I don't want to list everything.

Reply:
Yea you can take what I say or leave it.  Its an alpha feature so the amount of users modified container limts/requests will be very little.  Just want to bring up that this could become less true over time.

Reply:
Can you update this to say most at least?  

I think that over time this statement will become incorrect.

We could update this once in place pod becomes beta/ga if you want.

/lgtm

Reply:
I rephrased it.